recipes/ad3/recipe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ad3" %}
+{% set version = "2.5.3" %}
+{% set sha256 = "1a63e2e401a33467febc22ca10e91d7385c6813b2c3b44e96460360b76db147c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ad3-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/AD3/archive/{{ version }}.tar.gz
```

Comment:
So this appears to be a fork of an upstream project. Typically we try to avoid forking and push changes back to upstream if possible. What reasons are there for the fork?

That said, patching can be an option. Would that work in this case? Are there reasons it wouldn't work?